src/components/Calculator.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from 'react';
+import Button from './Button';
+import calculate from '../logic/calculate';
+
+class Calculator extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      operation: null,
+      total: 0,
+      next: null,
+    };
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick = (e) => {
+    this.setState((state) => calculate(state, e.target.innerText));
+  };
+
+  render() {
+    const { total, next, operation } = this.state;
+    return (
+      <div className="wrapper">
+        <div className="calculator">
+          <div className="screen">
+            {total}
+            {next}
+            {operation}
```

Comment:
- [ ] Please consider shifting the operation to the middle. If you realize most of calculators have their operation signs in the middle when computing, kindly consider applying that to ensure that the functionality gives a clear meaning 👍.
![image](https://github.com/Rifat8080/math-magicians/assets/99394374/f7ec9cb9-c8d7-4901-9ed7-df419a7bb29b)